iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/tree/KubernetesFileTree.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.kubernetes.tree;
+
+import java.util.List;
+import javax.annotation.Nullable;
+import org.sonar.iac.common.yaml.tree.FileTree;
+import org.sonar.iac.common.yaml.tree.FileTreeImpl;
+import org.sonar.iac.common.yaml.tree.YamlTree;
+import org.sonar.iac.common.yaml.tree.YamlTreeMetadata;
+import org.sonar.iac.helm.tree.Tree;
+
+public class KubernetesFileTree extends FileTreeImpl {
```

Comment:
I guess that there should be `api.KubernetesFileTree` and this class should be renamed to `KubernetesFileTreeImpl`